src/Views/Form-Themes/Sequoia/config.php

```diff
@@ -0,0 +1,127 @@
+<?php
+return [
+	'id'      => 'sequoia',
+	'name'    => __( 'Sequoia - Multi-Step Form', 'give' ),
+	'image'   => 'https://images.unsplash.com/photo-1448387473223-5c37445527e7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=100',
+	'options' => [
+		'introduction'        => [
+			'name'   => __( 'Introduction', 'give' ),
+			'desc'   => __( 'Step description will show up here if any', 'give' ),
+			'fields' => [
+				[
+					'id'         => 'headline',
+					'name'       => __( 'Headline', 'give' ),
+					'desc'       => __( 'Do you want to customize the headline for this form? We recommend keeping it to no more than 8 words as a best practive. If no title is provided the fallback will be your form’s post title.', 'give' ),
+					'type'       => 'text',
+					'attributes' => [
+						'placeholder' => __( 'Campaign Heading', 'give' ),
+					],
+				],
+				[
+					'id'         => 'description',
+					'name'       => __( 'Description', 'give' ),
+					'desc'       => __( 'Do you want to customize the description for this form? The description displays below the headline. We recommend keeping it to 1-2 short sentences. If no description is provided the fallback will be your form’s excerpt.', 'give' ),
+					'type'       => 'textarea',
+					'attributes' => [
+						'placeholder' => __( 'Help provide education, care, and community development. It couldn’t happen without you.', 'give' ),
+					],
+				],
+				[
+					'id'         => 'image',
+					'name'       => __( 'Image', 'give' ),
+					'desc'       => __( 'Upload an eye-catching image that reflects your cause. The image is required and if none is provided the featured image will be a the fallback. If none is set you will see a placeholder image displayed on the form. For best results use an image that’s 600x400 pixels.', 'give' ),
+					'type'       => 'file',
+					'query_args' => [
```

Comment:
@henryholtgeerts You can remove this param. it is not supported by the `file` or `media` field. We are handling that internally.
ref: https://github.com/impress-org/givewp/blob/54151c5f09f3145e17a0fdb0558c2c859be80fd3/includes/admin/give-metabox-functions.php#L885

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/91bbb4a8e9265e3fca9cb6e01c2361e80d427480